docs/mobile-apps/features/ios-version-management.md

```diff
@@ -0,0 +1,322 @@
+:::
+
+<Tabs
+groupId="capability-ex-emusim"
+defaultValue="java"
+values={[
+{label: 'Java', value: 'java'},
+{label: 'Node.js', value: 'js'},
+{label: 'Python', value: 'python'},
+{label: 'Ruby', value: 'ruby'},
+{label: 'C#', value: 'csharp'},
+]}>
+
+<TabItem value="java">
+<Tabs
+groupId="capability-java-emusim"
+defaultValue="iphone"
+values={[
+{label: 'iPhone', value: 'iphone'},
+{label: 'iPad', value: 'ipad'},
+]}>
+<TabItem value="iphone">
+
+<!-- prettier-ignore -->
+```java
+MutableCapabilities capabilities = new MutableCapabilities();
+
+capabilities.setCapability("browserName", "safari");
+capabilities.setCapability("platformName", "ios");
+// Use one of the following aliases to test on the latest version of iOS of your choice
+capabilities.setCapability("appium:platformVersion", "latest|current_major|previous_major");
+capabilities.setCapability("appium:deviceName", "iPhone Simulator");
+capabilities.setCapability("appium:automationName", "xcuitest");
+
+HashMap<String, Object> sauceOptions = new HashMap<String, Object>();
+capabilities.setCapability("sauce:options", sauceOptions);
+```
+
+</TabItem>
+<TabItem value="ipad">
+
+```java
+MutableCapabilities capabilities = new MutableCapabilities();
+
+capabilities.setCapability("browserName", "safari");
+capabilities.setCapability("platformName", "ios");
+// Use one of the following aliases to test on the latest version of iOS of your choice
+capabilities.setCapability("appium:platformVersion", "latest|current_major|previous_major");
+capabilities.setCapability("appium:deviceName", "iPad Simulator");
+capabilities.setCapability("appium:automationName", "xcuitest");
+
+HashMap<String, Object> sauceOptions = new HashMap<String, Object>();
+capabilities.setCapability("sauce:options", sauceOptions);
+```
+
+</TabItem>
+</Tabs>
+</TabItem>
+<TabItem value="js">
+
+<Tabs
+groupId="capability-js-emusim"
+defaultValue="iphone"
+values={[
+{label: 'iPhone', value: 'iphone'},
+{label: 'iPad', value: 'ipad'},
+]}>
+<TabItem value="iphone">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+    browserName: 'safari',
+    platformName: 'ios',
+    // Use one of the following aliases to test on the latest version of iOS of your choice
+    'appium:platformVersion': 'latest|current_major|previous_major',
+    'appium:deviceName': 'iPhone Simulator',
+    'appium:automationName': 'xcuitest',
+    'sauce:options': {
+    }
+}
+```
+
+</TabItem>
+<TabItem value="ipad">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+    browserName: 'safari',
+    platformName: 'ios',
+    // Use one of the following aliases to test on the latest version of iOS of your choice
+    'appium:platformVersion': 'latest|current_major|previous_major',
+    'appium:deviceName': 'iPad Simulator',
+    'appium:automationName': 'xcuitest',
+    'sauce:options': {
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem value="python">
+
+<Tabs
+groupId="capability-python-emusim"
```

Comment:
Does not render as proper md file, is this an error, or this will be used by SL help website UI to format it properly?

Reply:
This will be rendered properly by the [Docusaurus](https://docusaurus.io/)